integration/insert_run_command_compat_test.go

```diff
@@ -84,12 +107,8 @@ func TestInsertRunCommandCompat(t *testing.T) {
 			documents: []any{
 				bson.D{{"_id", "foo"}},
 			},
-			ordered: "foo",
-
-			// TODO: to be enabled in a separate PR of the same issue as some refactoring is needed.
-			// Compat error: "BSON field 'insert.ordered' is the wrong type 'string'
-			// Target error: "BSON field 'ordered' is the wrong type 'string'
-			skip: "https://github.com/FerretDB/FerretDB/issues/940",
+			ordered:     "foo",
+			altErrorMsg: "BSON field 'ordered' is the wrong type 'string', expected type 'bool'",
```

Comment:
Are we going to track this on dance?

Reply:
I don't think so :) We have a similar story for `findAndModify`(see `TestFindAndModifyErrors`), and it doesn't seem to be critical. Or at least, it doesn't seem to be critical at the moment.

I'd say the main point is that we validate wrong params and return a reasonable message.